content/en/training/_index.html

```diff
@@ -0,0 +1,107 @@
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Introduction to Linux</b>
+        </h5>
+        <p>Never learned Linux? Want a refresh? Develop a good working knowledge of Linux using both the graphical interface and command line across the major Linux distribution families.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-linux" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div class="padded lighter-gray-bg">
+  <div class="main-section two-thirds-centered">
+    <center>
+      <h2>Learn with the Linux Foundation</h2>
+      <p>The Linux Foundation offers instructor-led and self-paced courses for all aspects of the Kuberenetes application development and operations lifecycle.</p>
+      <br><br>
+      <a href="https://training.linuxfoundation.org/training/course-catalog/?_sft_technology=kubernetes" target="_blank" class="button">See Courses</a>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section padded">
+    <center>
+        <h2>Get Kubernetes Certified</h2>
+    </center>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Certified Kubernetes Application Developer (CKAD)</b>
+        </h5>
+        <p>The Certified Kubernetes Application Developer exam certifies that users can design, build, configure, and expose cloud native applications for Kubernetes.</p>
+        <br>
+        <a href="https://training.linuxfoundation.org/certification/certified-kubernetes-application-developer-ckad/" target="_blank" class="button">Go to Certification</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Certified Kubernetes Administrator (CKA)</b>
+        </h5>
+        <p>The Certified Kubernetes Administrator (CKA) program provides assurance that CKAs have the skills, knowledge, and competency to perform the responsibilities of Kubernetes administrators.</p>
+        <br>
+        <a href=https://training.linuxfoundation.org/certification/certified-kubernetes-administrator-cka/" target="_blank" class="button">Go to Certification</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div class="padded lighter-gray-bg">
+  <div class="main-section two-thirds-centered">
+    <center>
+      <h2>Kubernetes Training Partners</h2>
+      <p>Our network of Kubernetes Training Parnters provide training and consulting services for Kubernetes and cloud native projects.</p>
+    </center>
+  </div>
+  <div class="main-section">
+    <iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no" style="opacity: 1; visibility: visible; overflow: hidden; height: 1200px; width: 100%"></iframe>
```

Comment:
One more suggestion (also from Chris A): a script to resize the partners table in a more friendly way, like the [CNCF training page](https://www.cncf.io/certification/training/):

```suggestion
    <iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no" style="opacity: 1; visibility: visible; overflow: hidden; height: 1200px; width: 100%"></iframe>
    <script src="https://landscape.cncf.io/iframeResizer.js"></script>
```